panacea/tx.go

```diff
@@ -20,6 +21,20 @@ func NewTxBuilder(client QueryClient) *TxBuilder {
 	}
 }
 
+// GenerateTxBytes generates transaction byte array.
+func (tb TxBuilder) GenerateTxBytes(privKey cryptotypes.PrivKey, conf *config.Config, msg ...sdk.Msg) ([]byte, error) {
```

Comment:
I have a question. Why we use the `msg` data type `...sdk.Msg` instead of `sdk.Msg`?

Reply:
`TxBuilder` in `cosmos-sdk/client` interface define `SetMsgs(msgs ...sdk.Msg)` that can be set multi Msgs and multi signatures.  I think we use this interface to generate signed transaction, so we just follow this form.

Reply:
Ah, I got it. Thanks for explain!

Reply:
In general, several `sdk.Msg`s(actions) can be included in a single transaction.
So the `TxBuilder` get multiple `sdk.Msg`s.